prow/plugins/trigger/pull-request.go

```diff
@@ -106,6 +106,45 @@ func handlePR(c Client, trigger plugins.Trigger, pr github.PullRequestEvent) err
 				return buildAll(c, &pr.PullRequest, pr.GUID, trigger.ElideSkippedContexts)
 			}
 		}
+		if pr.Label.Name == labels.OkToTest {
+			labelAddedBy := pr.Sender.Name
+			// If label is added by the bot, then skip it.
+			// this could also be possible if someone goes the traditional way of adding /ok-to-test then /test-all comments
+			// to trigger the testing
+			botName, err := c.GitHubClient.BotName()
+			if err != nil {
+				return err
+			}
+			if labelAddedBy == botName {
+				c.Logger.Debug("Label added by the bot, skipping.")
+				return nil
+			}
+			trusted, err := TrustedUser(c.GitHubClient, trigger.OnlyOrgMembers, trigger.TrustedOrg, labelAddedBy, org, repo)
```

Comment:
I wonder if we need this check, in the case of the LGTM label we do not check if the one who added it is a member of the org